docs/csharp/language-reference/compiler-messages/cs8400.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Compiler Error CS8400
+ms.date: 07/11/2020
+f1_keywords:
+  - "CS8400"
+helpviewer_keywords:
+  - "CS8400"
+author: Youssef1313
+---
+# Compiler Error CS8400
+
+Feature 'feature' is not available in C# 8.0. Please use language version \<version> or greater.
```

Comment:
```suggestion
Feature is not available in C# 8.0. Please use language version \<version> or greater.
```

Reply:
@IEvangelist I think other existing error messages keeps placeholders like this.